src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Ccda/ReferralNote/sample-expected.json

```diff
@@ -274,9 +272,7 @@
             "language": {
               "coding": [
                 {
-                  "system": "urn:ietf:bcp:47",
-                  "code": "fr-CN",
-                  "display": "fr-cn"
+                  "system": "urn:ietf:bcp:47"
```

Comment:
Why no code?

Reply:
the code `fr-cn` isn't a valid language code as defined [here](https://www.hl7.org/fhir/valueset-languages.html), for this case, an empty string will be generated and be removed in the postprocessing.